examples/vic-gov-au/nuxt.config.js

```diff
@@ -143,12 +153,7 @@ export default {
       webform: 1,
       search: 1,
       authenticatedContent: 1,
-      alert: 1,
-      gtm: 1
```

Comment:
Did we ever switch this off before for specific site?

Reply:
I think there are two sites have turned it off, but it should be ok. Let them manually update this.